projects/hslayers/src/components/layermanager/layermanager-metadata.service.ts

```diff
@@ -63,7 +73,7 @@ export class HsLayerManagerMetadataService {
   }
 
   metadataArray(layer: Layer): Array<any> {
-    const obj = layer.layer.get('MetadataURL');
```

Comment:
I'm afraid this wont be so easy. Should explore how the MetadataURL which is a string is used. 'metadata' is an object which can hold anything currently. We could have layer.get('metadata') be an object {key: string, key2: string, url: string} where url was MetadataURL, but have to be reasonably sure we dont break anything.

Reply:
Turns out MetadataURL is a aray-ish object similar to {'0': {OnlineResource: 'http://..'}}. I Rewrote it to be incorporated into the metadata attribute which is now a more complex object of {urls: [], style: {}}